ante/spamming_prevention.go

```diff
@@ -0,0 +1,108 @@
+package ante
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	oracletypes "github.com/umee-network/umee/x/oracle/types"
+)
+
+// SpammingPreventionDecorator will check if the transaction's gas is smaller than
+// configured hard cap.
+type SpamPreventionDecorator struct {
+	oracleKeeper     OracleKeeper
+	oraclePrevoteMap map[string]int64
+	oracleVoteMap    map[string]int64
```

Comment:
what happened to the mutex?

Reply:
I took it out, just misunderstood your comment, my bad